core/src/main/java/io/grpc/MessageEncoding.java

```diff
@@ -17,11 +21,13 @@
 /**
  * Encloses classes related to the compression and decompression of messages.
  */
+@ExperimentalApi
 public final class MessageEncoding {
   /**
    * Special sentinel codec indicating that no compression should be used.  Users should use
    * reference equality to see if compression is disabled.
    */
+  @ExperimentalApi
```

Comment:
In fact, would it make sense to have this as a static method on `Codec` (e.g. `Codec.none()` or `Codec.identity()`)?